chalicelib/services/BroadcastService.py

```diff
@@ -57,35 +78,215 @@ def generate_newsletter_content(self, custom_content: str = "") -> Dict:
         consulting_section = self.generate_section_html("Consulting", consulting_jobs)
         marketing_section = self.generate_section_html("Marketing", marketing_jobs)
         
-        # Common CSS for styling the newsletter sections
+        # Updated CSS for styling the newsletter to match the provided image
         css = """
```

Comment:
Is there a way to modularize or put this into utils where you can call that constant? This takes way too much space and it's hard to understand the core functionality.